core/unit_test/tstParameterPack.hpp

```diff
@@ -28,14 +26,14 @@ void captureTest()
     Kokkos::View<int[1][1], TEST_MEMSPACE> int_view( "int_view" );
 
     // Make a parameter pack so we can capture them as a group.
-    auto pack = makeParameterPack( dbl_view, int_view );
+    auto pack = Cabana::makeParameterPack( dbl_view, int_view );
 
     // Update the pack in a kernel
     Kokkos::parallel_for(
         "fill_pack", Kokkos::RangePolicy<TEST_EXECSPACE>( 0, 1 ),
         KOKKOS_LAMBDA( const int ) {
-            auto dv = get<0>( pack );
-            auto iv = get<1>( pack );
+            auto dv = Cabana::get<0>( pack );
```

Comment:
(Not asking you to change just making sure you know) I expect you don't absolutely have to qualify the call, that the function would be found by ADL (argument-dependent lookup)